shared/components/AppBar.jsx

```diff
@@ -34,12 +35,13 @@ export default class AppBar extends React.Component {
         isFixedToTop : false
     };
 
-    handleScroll = (e) => {
-        const scrollTop = event.srcElement.body.scrollTop;
+    handleScroll = () => {
+        const scrollTop = $(window).scrollTop();
```

Comment:
does it so necessary to import jQuery only for [this](http://stackoverflow.com/questions/3464876/javascript-get-window-x-y-position-for-scroll)? btw, it'll be loaded while ssr also ;)